arduino-ide-extension/src/browser/widgets/component-list/list-item-renderer.tsx

```diff
@@ -1,137 +1,779 @@
-import * as React from '@theia/core/shared/react';
-import { inject, injectable } from '@theia/core/shared/inversify';
+import { ApplicationError } from '@theia/core';
+import {
+  Anchor,
+  ContextMenuRenderer,
+} from '@theia/core/lib/browser/context-menu-renderer';
+import { TabBarToolbar } from '@theia/core/lib/browser/shell/tab-bar-toolbar';
+import { codicon } from '@theia/core/lib/browser/widgets/widget';
 import { WindowService } from '@theia/core/lib/browser/window/window-service';
-import { Installable } from '../../../common/protocol/installable';
-import { ArduinoComponent } from '../../../common/protocol/arduino-component';
-import { ComponentListItem } from './component-list-item';
-import { nls } from '@theia/core/lib/common';
+import {
+  CommandHandler,
+  CommandRegistry,
+  CommandService,
+} from '@theia/core/lib/common/command';
+import {
+  Disposable,
+  DisposableCollection,
+} from '@theia/core/lib/common/disposable';
+import {
+  MenuModelRegistry,
+  MenuPath,
+  SubMenuOptions,
+} from '@theia/core/lib/common/menu';
+import { MessageService } from '@theia/core/lib/common/message-service';
+import { nls } from '@theia/core/lib/common/nls';
+import { inject, injectable } from '@theia/core/shared/inversify';
+import * as React from '@theia/core/shared/react';
 import { Unknown } from '../../../common/nls';
+import {
+  CoreService,
+  ExamplesService,
+  LibraryPackage,
+  Sketch,
+  SketchContainer,
+  SketchesService,
+  SketchRef,
+} from '../../../common/protocol';
+import type { ArduinoComponent } from '../../../common/protocol/arduino-component';
+import { Installable } from '../../../common/protocol/installable';
+import { openClonedExample } from '../../contributions/examples';
+import {
+  ArduinoMenus,
+  examplesLabel,
+  unregisterSubmenu,
+} from '../../menu/arduino-menus';
+
+const moreInfoLabel = nls.localize('arduino/component/moreInfo', 'More info');
+const otherVersionsLabel = nls.localize(
+  'arduino/component/otherVersions',
+  'Other versions'
```

Comment:
```suggestion
  'Other Versions'
```
The IDE's other context menus use title case for the menu items so I assume that is the standardized style.

Reply:
It's in lowercase in the design, but I agree to change it to uppercase.

https://github.com/arduino/arduino-ide/pull/1927/commits/8c973ab111d42dbbf4dca753d6f1658739d63de0